sztp-agent/pkg/secureagent/utils.go

```
@@ -94,8 +95,10 @@ func (a *Agent) doTLSRequest(input string, url string, empty bool) (*BootstrapSe
 func generateInputJSONContent() string {
 	osName := replaceQuotes(strings.Split(linesInFileContains(OS_RELEASE_FILE, "NAME"), "=")[1])
 	osVersion := replaceQuotes(strings.Split(linesInFileContains(OS_RELEASE_FILE, "VERSION"), "=")[1])
-
-	// dmidecode.Dmidecode(true))  // This is one possibility to get hw information
+	baseboard, err := ghw.Baseboard()
+	if err != nil {
+		fmt.Printf("Error getting baseboard info: %v", err)
```

@alknopfler should we return an error here ?